packages/typography/src/TypographyBase.tsx

```diff
@@ -125,7 +144,7 @@ const responsiveStyles = {
  * Gets responsive styles.
  */
 function getResponsiveCss(props: TypographyProps) {
-  if (!props.variant) {
+  if (!props.variant || typeof props.variant !== 'string') {
```

Comment:
This function changes styling based on the screen size. We decided to remove it at all, but later as a tech debt. For now it will be only disabled when variant prop is an object, so it doesn't break breakpoint styles.